.gitignore

```diff
@@ -1,6 +1,6 @@
 # ipynb and csv
 *.ipynb
-*.csv
+eight_moods.csv
```

Comment:
I need the file locally to retrain the model every time. Doesn't affect the team repo.